d3a_interface/constants_limits.py

```diff
@@ -76,14 +77,16 @@ class StorageSettings:
         MAX_ABS_POWER_RANGE = RangeLimit(0.0001, 2000000)
         # Energy buy-range, storage never buys outside this limit.
         # Unit is ct/kWh.
-        BUYING_RANGE = RangeLimit(0, 24.9)
-        INITIAL_BUYING_RANGE = RangeLimit(0, 10000)
-        FINAL_BUYING_RANGE = RangeLimit(0, 10000)
+        INITIAL_BUYING_RATE = 0
```

Comment:
you could keep the BUYING_RATE variable and just assign a RateRange to it. That way you could access its parameters like BUYING_RATE.initial and BUYING_RATE.final, which is much more readable and easily maintainable IMO compared to 2 separate variables.

Reply:
Maybe I am to blame here because i might have confused Faizan.
My proposal was: Create a Raterange namedtuple with "intital" and "final" memebers and assign it to only `BUYING_RANGE` and `SELLING_RANGE` . The other limits should be a Limitrange or as you called it a Rangelimit with members "min" and "max". I hope this is clear now.

Reply:
Okay, to be 100% clear this this time and to be on the same page. Following changes needs to be done:

- Replace `INITIAL_BUYING_RATE` & `FINAL_BUYING_RATE` with `BUYING_RATE_RANGE = RateRange(0, 24.9)`.
- Replace `INITIAL_SELLING_RATE` & `FINAL_SELLING_RATE` with `SELLING_RATE_RANGE = RateRange(30, 25)`